columnflow/calibration/cms/jets.py

```diff
@@ -563,6 +563,8 @@ def get_jer_config(self) -> DotDict:
     propagate_met=True,
     # only run on mc
     mc_only=True,
+    # use deterministic seeds for random smearing
+    use_deterministic_seeds=False,
```

Comment:
I forgot...we need to have the index (e.g. the *i*th random number to use per seed) configurable, and we can use the same index to en/disable to use of deterministic seeds.

```suggestion
    # use deterministic seeds for random smearing and
    # take the "index"-th random number per seed when not -1
    deterministic_seed_index=-1,
```